Formula/p/podman-tui.rb

```diff
@@ -0,0 +1,26 @@
+class PodmanTui < Formula
+  desc "Podman Terminal User Interface"
+  homepage "https://github.com/containers/podman-tui"
+  url "https://github.com/containers/podman-tui.git",
+      tag:      "v0.17.0",
+      revision: "b4350927babdaa35fbf50424190683aeb9f5f66f"
+  license all_of: ["Apache-2.0", "GPL-3.0-or-later"]
+  head "https://github.com/containers/podman-tui.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    if OS.mac?
+      system "make", "binary-darwin"
+      bin.install "bin/darwin/podman-tui" => "podman-tui"
+    else
+      system "make", "binary"
+      bin.install "bin/podman-tui" => "podman-tui"
+    end
```

Comment:
better to use `std_go_args` build

Reply:
not sure if I can use "std_go_args", it looks like its useful for go ldflags however I am using build flags in the Makefile.

Reply:
let me check on the std_go_args thing really quick.

Reply:
Thanks @chenrui333